oauth2client/contrib/gce.py

```diff
@@ -36,10 +35,7 @@
 logger = logging.getLogger(__name__)
 
 # URI Template for the endpoint that returns access_tokens.
-_METADATA_ROOT = ('http://metadata.google.internal/computeMetadata/v1/'
-                  'instance/service-accounts/default/')
-META = _METADATA_ROOT + 'token'
-_DEFAULT_EMAIL_METADATA = _METADATA_ROOT + 'email'
+_METADATA_ROOT = 'http://metadata.google.internal/0.1/meta-data'
```

Comment:
Also, it unfortunately looks like the *removed* support for `expires_at` going from 0.1 to v1

>_<